src/domain/ecoverse/ecoverse.resolver.queries.ts

```diff
@@ -89,6 +89,25 @@ export class EcoverseResolverQueries {
     throw new Error(`Unable to locate user with given id: ${id}`);
   }
 
+  @Roles(
+    RestrictedGroupNames.CommunityAdmins,
+    RestrictedGroupNames.EcoverseAdmins
+  )
+  @UseGuards(GqlAuthGuard)
+  //should be in user queries
+  @Query(() => [User], {
+    nullable: false,
+    description: 'A particular user, identified by the ID or by email',
```

Comment:
Change the description - this is misleading and is the actual API schema documentation that is visible to gql consumers.